### PR TITLE
[zh] Fix a docs formatting error of Kubernetes Gateway API

### PR DESCRIPTION
**Pull request by nicole-lihui**
## Description

 Fix a docs formatting error of Kubernetes Gateway API, Ref: https://github.com/istio/istio.io/issues/14680
 
## Reviewers

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [X] Localization/Translation

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 1]
```diff
@@ -48,11 +48,11 @@ Gateway API 与 Istio API（如 Gateway 和 VirtualService）有很多相似之
 以构建标准化的，独立于供应商的 API。
 这些 API 通常与 Istio Gateway 和 VirtualService 具有相同的用途，但有一些关键的区别：
 
-* Istio API 中的 `Gateway` 仅配置已部署的现有网关 Deployment/Service，
-  而在 Gateway API 中的 `Gateway` 资源不仅配置也会部署网关。
+* Istio API 中的 `Gateway` *仅配置已部署*的现有网关 Deployment/Service，
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:51]:
```suggestion
* Istio API 中的 `Gateway` **仅配置**[已部署](/zh/docs/setup/additional-setup/gateway/)的现有网关 Deployment/Service，
```

File: content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md [hunk 2]
```diff
@@ -48,11 +48,11 @@ Gateway API 与 Istio API（如 Gateway 和 VirtualService）有很多相似之
...
+  而在 Gateway API 中的 `Gateway` 资源*不仅配置也会部署*网关。
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/ingress/gateway-api/index.md:52]:
```suggestion
  而在 Gateway API 中的 `Gateway` 资源**不仅配置也会部署**网关。
```

**Review by windsonsea — COMMENTED** [review]
中文建议改为 粗体，不要用 斜体，因为中文是方块字

**Comment by nicole-lihui** [comment]
> 中文建议改为 粗体，不要用 斜体，因为中文是方块字，歪斜很不常见

确实

**Review by windsonsea — APPROVED** [review]
Thanks

**Pull request merged**